src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -5548,6 +5548,40 @@ int32_t NaClSysAccess(struct NaClAppThread *natp,
   return ret;
 }
 
+int32_t NaClSysTruncate(struct NaClAppThread *natp,
+                        uint32_t file, int length) {
```

Comment:
There's a comment in syscall_handlers_gen.py that says you should use a uint32_t but we've mostly been ignoring that. I think it makes more sense to follow that because we're really only passing 32 bits of stuff into NaCl here.